_security/access-control/permissions.md

```diff
@@ -65,6 +65,18 @@ Rather than individual permissions, you can often achieve your desired security
 {: .tip }
 
 
+## System permission
+
+The system permission `system:admin/<system_index_name>` is unique among other permissions in that it extends some traditional admin-only accessibility to non-admin users. This permission gives normal users the ability to modify the system index specified in the permission name. For example, the permission `system:admin/.opendistro-alerting-config` gives a user permission to modify the system index that stores configurations for the Alerting plugin. 
+
+The system permission excludes, however, access to the security system index `.opendistro_security`, which is used to store Security's configuration YAML files and remains accessible only to admins with an admin certificate.
```

Comment:
Thoughts on: `The system permission, however, excludes access to the `.opendistro_security` system index, which stores the Security plugin's configuration, and is only accessible to admins with an admin certificate.`